lib/Common/ConfigFlagsList.h

```diff
@@ -626,12 +626,16 @@ PHASE(All)
 #define DEFAULT_CONFIG_ES6String               (true)
 #define DEFAULT_CONFIG_ES6StringPrototypeFixes (true)
 #define DEFAULT_CONFIG_ES2018ObjectSpread      (false)
+
+#ifndef DEFAULT_CONFIG_ES6PrototypeChain
 #ifdef COMPILE_DISABLE_ES6PrototypeChain
```

Comment:
I think we can get rid of this COMPILE_DISABLE* for this one as this is enabled now.